pkg/queue/postgres/fixtures_test.go

```diff
@@ -9,3 +15,12 @@ var (
 	progress = []byte(`{"scale": 99}`)
 	spec     = []byte(`{"field": "value"}`)
 )
+
+
+func verifyLeak(t *testing.T) {
+	_, exists := os.LookupEnv("CI")
+	if exists {
+		return
+	}
+	goleak.VerifyNone(t)
+}
```

Comment:
why do we need it defined twice? there is a copy in `api_request_test.go`.

Reply:
I didn't want to define it as public method